### DISCUSSION TITLE
圧力陰解法時のrelaxation coefficientって？

### DISCUSSION THREAD

**Discussion by iori3150** [MPS-Basic/MPS-Basic#146] [Q&A] [answered]
なぜ必要で、それは物理的にはどのような意味を持つのでしょうか？

**Comment by zukain5** [comment]
@Quesys-tech

**Comment by Quesys-tech** [comment, marked as answer]
- 計算の安定のために必要です。圧力Poisson方程式は緩和がないと離散化の誤差などで極端な値が出てしまいます
- 物理的な意味はあまりないと思います。

**Comment by iori3150** [comment]
えーーー、物理的な意味がないことはあまりやりたくない心境なんですが

**Reply by iori3150** [reply]
圧力値に影響出ないんでしょうか
